backend/src/routes/calendlyPage.ts

```diff
@@ -0,0 +1,62 @@
+import express, { NextFunction, Request, Response } from "express";
+
+const router = express.Router()
+
+// Add Firebase Auth
+router.get('/calendly', (req: Request, res: Response) => {
+    const calendlyLink = req.query.url;
+    const calendlyPage = `
+    <html>
+    <head>
+        <meta name='viewport' content='width=device-width' />
+    </head>
+    <script>
+        function sendMessage(message) {
+            webkit.messageHandlers.calendlyURI.postMessage(message);
+            console.log("message sent", window.parent.origin);
+            fetch('http://127.0.0.1:5000/')
```

Comment:
Let's make this URL a constant as well and after deployment thsi needs to be set to the URL for our backend